zwave_js_server/model/value.py

```diff
@@ -170,3 +171,32 @@ def receive_event(self, event: Event) -> None:
         """Receive an event."""
         self.data.update(event.data["args"])
         self._value = event.data["args"].get("newValue")
+
+
+@dataclass
+class ValueNotification:
+    """
+    Model for a Value Nofification message.
+
+    https://zwave-js.github.io/node-zwave-js/#/api/node?id=quotvalue-notificationquot
+    """
+
+    command_class_name: str
+    command_class: int
+    endpoint: int
+    property: str
+    value: Any
+    property_name: str
+    metadata: Optional[ValueMetadata] = None
+
+    @classmethod
+    def from_event(cls, event: Event):  # type: ignore
```

Comment:
```suggestion
    def from_event(cls, event: Event) -> "ValueNotification":
```

Reply:
Tried that, mypy was still complaining. Maybe that was just me being blind. I'll try again